packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2074,6 +2093,14 @@ class InputDecoration {
   /// See [Icon], [ImageIcon].
   final Widget suffixIcon;
 
+  /// Optional widget to place on the line after the input.
+  /// Can be for example used to add some padding to the [suffixText] or to
+  /// add a custom widget after the input.
+  ///
+  /// Either [suffix] or [suffixText] can be declared.
```

Comment:
Only one of [suffix] and [suffixText] can be specified.

You can leave the "Declaring both" statement out.